src/expo-plugins/withNotificationControls.ts

```diff
@@ -24,6 +24,15 @@ export const withNotificationControls: ConfigPlugin<boolean> = (
         application.service = [];
       }
 
```

Comment:
@freeboub
I think it's better to add a log to explain this logic.
```suggestion
// We check if the VideoPlaybackService is already defined in the AndroidManifest.xml
// to prevent adding duplicate service entries. If the service exists, we will remove 
// it before adding the updated configuration to ensure there are no conflicts or redundant
// service declarations in the manifest.
```
WDYT?